AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -345,6 +352,16 @@
         ]
       }
     },
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
+      "apiVersion": "2022-04-01",
+      "name": "[parameters('roleNameGuid')]",
+      "scope": "[parameters('azureMonitorWorkspaceResourceId')]",
+      "properties": {
+          "roleDefinitionId": "[concat('/subscriptions/', variables('clusterSubscriptionId'), '/providers/Microsoft.Authorization/roleDefinitions/', 'acdd72a7-3385-48ef-bd42-f606fba81ae7')]",
```

Comment:
Please update this to the GUID for the Monitoring Data Reader Role.

Reply:
Sure. Thanks!